ffi/java-idsdk/README.md

```diff
@@ -0,0 +1,54 @@
+# Java wrapper for the ID SDK
+
+FFI wrapper using JNA
+
+## Build
+
+`$> mvn pakage`
+
+## Test
+
+A sample app is in the test/java directory: `com.iotics.sdk.identity.App`
+
+## Use
+
+```java
+SdkApi api = new JnaSdkApiInitialiser("<path_to>/lib-iotics-id-sdk-amd64.so").get();
```

Comment:
if you have the library somewhere else and you want to specify the path to it.
it's easier in the current layout to build the `so` in the ffi/ directory and refer to it like in the App.java file as `"../bin/...so"`.

Reply:
The current paths are fine.
I'm just asking if there's a use case for the .so being in a different path other than the default.
If not, it would be best to remove the constructor that allows a custom path, in order to simplify the API.